code/modules/mob/living/simple_animal/animals/farm_animals.dm

```diff
@@ -155,7 +155,7 @@
 /mob/living/simple_animal/chick
 	name = "\improper chick"
 	desc = "Adorable! They make such a racket though."
-	tt_desc = "Gallus domesticus"
+	tt_desc = "E Gallus gallus domesticus"
```

Comment:
the 'domesticus' subspecies is generally less recognized and used less commonly each year